README.md

```diff
@@ -72,7 +72,10 @@ $ git submodule update --init --force
 After getting the submodule path setup, copy the `arduino-base.ino` file from the `arduino-base` submodule to your `Arduino` directory and rename the new copy to `Arduino.ino`. Use this file as a starting point to start working on your project. Inside the sketch are comments that explain how to use the libraries and how you can use the serial messaging system for parsing and sending messages.
 
 #### Setting up an app
-Finally, after following the setup above, you'll need to register an `ipcRenderer` from `Electron` to allow stele and `arduino-base` to communicate. If you're using the SMM `create-react-app`, this can be accomplished by adding a reference in `public/index.html` file.
+Finally, after following the setup above, you'll need to register an `ipcRenderer` from `Electron` to allow stele and `arduino-base` to communicate. 
```

Comment:
@tnordberg and I discussed that there should be a little more documentation here to make sure developers understand exactly what to do.

Reply:
Instructions added.